docs/products/postgresql/howto/pagila.rst

```diff
@@ -0,0 +1,251 @@
+Pagila – a PostgreSQL sample database
+=====================================
+
+Aiven provides a sample database you can import in your Aiven for PostgreSQL service. This page covers information about the database and the procedure to get it up and running.
+
+Pagila is a port of the `Sakila Sample Database <https://dev.mysql.com/doc/sakila/en/>`_. It was originally developed by Mark Hillyer, a former member of the MySQL AB documentation team, to provide a standard schema and data for articles, tutorials, books etc. Over time, it has been ported by the various developers to `multiple database systems <https://github.com/search?q=sakila>`_. The examples here use one from ``devrimgunduz``, `version 2.1.0 <https://github.com/devrimgunduz/pagila>`_.
+
+Sakila (and thus, Pagila) is a database representing a DVD rental store (remember those?!), containing information about films (like title, category, actresses), rental stores (like address, staff members, customers) and rentals, where a customer rents a film from a store through its staff.
+
+With all these relational information, Pagila is a perfect fit to play around with PostgreSQL and the SQL language.
+
+Load Pagila on PostgreSQL
+-------------------------
+
+Before exploring the Pagila database, we need a PostgreSQL server to load the data on. A quick and easy way to get a server is through `Aiven for PostgreSQL <https://aiven.io/postgresql>`_. If don't have an Aiven account yet, `sign up <https://console.aiven.io/signup?utm_source=github&amp;utm_medium=organic&amp;utm_campaign=devportal&amp;utm_content=repo>`_ and enjoy our free trial!
```

Comment:
We can try to just say that you need a running PostgreSQL service.

Reply:
Ajusted.